docs/Setup.md

```diff
@@ -11,7 +11,7 @@ using the default integration provided by mermaid.js.
 The core of this api is the [**render**][2] function which, given a graph
 definition as text, renders the graph/diagram and returns an svg element for the graph.
 
-It is is then up to the user of the API to make use of the svg, either insert it somewhere in the
+It is then up to the user of the API to make use of the svg, either insert it somewhere in the
```

Comment:
Setup.md is auto generated with `yarn build`.
The correct place to change this would be in `src/mermaidAPI.js`.

Reply:
I've extracted the doc typo removal into [a separate PR](https://github.com/mermaid-js/mermaid/pull/3403)